libraries/common/stream.effekt

```diff
@@ -0,0 +1,417 @@
+module stream
+
+import array
+import bytearray
+import io/filesystem
+import io/error
+
+
+effect emit[A](value: A): Unit
```

Comment:
I would personally prefer this to be Uppercase for consistency with the other effects I've come across...